docs/getstarted/getting-started.md

```diff
@@ -0,0 +1,277 @@
+
+<iframe src="https://www.youtube.com/embed/B-s71n0dHUk?autoplay=true" width="640" height="320" allowFullScreen="true" frameBorder="0" title="Getting Started with Visual Studio Code"></iframe>
+
+## Prerequisites
+
+- [Download Visual Studio Code](https://code.visualstudio.com/download)
+- [Set up Visual Studio Code on your computer](/docs/setup/setup-overview.md)
+- [Install Node.js](https://nodejs.org/)
+
+## Step 1: Open a folder in VS Code
+
+VS Code is a great lightweight editor where you can with individual files seamlessly, for example for quick edits or to view a file. However, by opening a folder, also known as a *workspace*, you unlock more features, such as folder-specific configuration settings, restoring the UI state for that folder, debug configurations, and more. Get more info about [workspaces](/docs/editor/workspaces.md).
+
+Let's start by creating a folder and opening it in VS Code. You'll use this folder throughout the tutorial.
+
+1. Create a new folder `vscode101` on your computer.
+
+1. Open Visual Studio Code.
+
+    When you first open VS Code, you should see the **Welcome** page with different actions to get started. You can also explore some of the VS Code product walkthroughs.
+
+1. On the Welcome page, select **Open Folder...**, and then select the folder you created.
+
+    The VS Code window reloads and you should see the folder name at the top of the **Explorer** view. You'll use the Explorer view to view and manage the files and folders in your workspace.
+
+    ![Screenshot that shows VS Code after opening a folder, highlighting the Explorer view.](images/getting-started/open-folder.png)
+
+    Alternatively, you can select **Open Folder...** from the **File** menu or press `kb(workbench.action.files.openFolder)` to open a folder.
+
+> **Tip**: When you use the command line, pass the folder name as a parameter to directly launch VS Code for that folder. For example, enter `code .` to open the current folder in VS Code.
+
+## Step 2: Explore the user interface
+
+Now that you have a folder open in VS Code, let's take a quick tour of the user interface.
+
+### Switch between views with the Activity Bar
+
+The **Activity Bar** is located on the side of the window and gives you quick access to different views, such as the **Explorer**, **Search**, **Source Control**, and **Run and Debug** views.
+
+1. Use the Activity Bar to explore the different views.
+
+    As you hover over the Activity Bar, you can see the name of each view and the keyboard shortcut to open each view. You'll find that many of the features in VS Code have keyboard shortcuts already assigned to them, and you can also customize these shortcuts to your liking.
+
+    ![Screenshot that highlights the Activity Bar.](images/getting-started/activity-bar.png)
```

Comment:
Love this screenshot. I think it's great you call out the hover info; I almost missed it in the screenshot, so I'd consider putting a red box around the hover info too.